### PR TITLE
added rhacs-central.json for in-cluster and cluster-wide central dasboards

### PR DESCRIPTION
**Pull request by vinaybommana**
Hey Team, for adding central dashboard to use rhobs metrics / cluster-wide dashboard, I've added the generated json structure for central dashboard. this will not affect the existing `rhacs-central-dashboard.yaml` grafana dashboard, but both in-cluster and cluster-wide dashboards will be generated from `rhacs-central.json`
Screenshot from 2023-08-04 11-59-07

this will proceed the consolidated dashboard work, I'll be merging the cluster-overview and slo dashboards by creating json and following the same structure

I'm having trouble parsing the `rhacs-central.json` using the command
```
yq ".spec.json = ($(cat generated/dashboards/rhacs-central.json) | to_json)" templates/dashboards/rhacs-central-dashboard.yaml
```
`yq` is throwing me errors that the argument list is too long, since the json is `5226` lines long, I'm not sure how to mitigate this so I've used `sed` and generated the yaml files in a hacky way, please provide any suggestions regarding this

as part of [ROX-18784](https://issues.redhat.com/browse/ROX-18784)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .PHONY: generate
 generate:
 	$(MAKE) -C resources/mixins/kubernetes generate
+	$(MAKE) -C resources/grafana generate
```
Comment (stehessel) [inline Makefile:4]: Hmm this will currently also update the Kubernetes mixin revision if one just wants to generate the dashboard. Although I guess the problem is more with the mixin generate command, we should separate it from the update part. But we can do that in a separate PR.

File: resources/grafana/generated/dashboards/rhacs-central.json [hunk 2]
```diff
@@ -0,0 +1,5229 @@
+{
```
Comment (stehessel) [inline resources/grafana/generated/dashboards/rhacs-central.json:1]: I think the file structure is a bit unfortunate as the `.json` here is the source of truth for the dashbord `.yaml`. So I would rather put the latter into a `/generated` folder and keep the `.json` more top level.

**Comment by stehessel** [comment]
> yq ".spec.json = ($(cat generated/dashboards/rhacs-central.json) | to_json)" templates/dashboards/rhacs-central-dashboard.yaml

This works fine for me. What version of yq are you using?

File: resources/grafana/rhacs-central-configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,5240 @@
+apiVersion: v1
```
Comment (stehessel) [inline resources/grafana/rhacs-central-configmap.yaml:1]: Maybe we should put this into a separate directory?

File: resources/grafana/templates/dashboards/rhacs-central-configmap.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```
Comment (stehessel) [inline resources/grafana/templates/dashboards/rhacs-central-configmap.yaml:1]: Let's add a comment to these template that explains why there is a `ConfigMap` and a `GrafanaDashboard` version.

**Comment by vinaybommana** [comment]
```
➜  grafana git:(ROX-18784) yq --version
yq (https://github.com/mikefarah/yq/) version v4.34.2
```
I'm facing issues only with very large json files, which in this case `rhacs-central.json` is

File: resources/grafana/generated/dashboards/rhacs-central-dashboard.yaml [hunk 5]
```diff
@@ -1,3 +1,4 @@
+# rhacs-central-dashboard.yaml is generated from rhacs-central.json
```
Comment (stehessel) [inline resources/grafana/generated/dashboards/rhacs-central-dashboard.yaml:1]: You have to change the location in the `index.json` as well after moving the file.

**Comment by stehessel** [comment]
> I'm facing issues only with very large json files, which in this case rhacs-central.json is

Strange, I cannot reproduce this with the same version. But I guess it's ok to use sed then.

File: resources/grafana/generated/dashboards/rhacs-central-dashboard.yaml (hunk 5 above)
Comment (vinaybommana) [reply resources/grafana/generated/dashboards/rhacs-central-dashboard.yaml:1]: Thank you @stehessel, updated the `index.json`, can you explain the process on how these dashboards will be deployed onto clusters

Comment (stehessel) [reply resources/grafana/generated/dashboards/rhacs-central-dashboard.yaml:1]: Sure, the observability operator (https://github.com/redhat-developer/observability-operator) has a pull mechanism which pulls the config from a configured git revision.

**Review by stehessel — APPROVED** [review]
LGTM, just need to fix the CI linter.
